python/cuml/svm/svm_base.pyx

```diff
@@ -541,27 +569,19 @@ class SVMBase(Base):
         state = self.__dict__.copy()
         del state['handle']
         del state['_model']
-
         # Only when the model is fit once we need to store these parameters
-        if self.fit_status_ == 0:
-            state['dual_coef_'] = \
-                cudf.DataFrame.from_gpu_matrix(self.dual_coef_)
-            state['support_'] = cudf.Series(self.support_)
-            state['support_vectors_'] = \
-                cudf.DataFrame.from_gpu_matrix(self.support_vectors_)
-            state['_unique_labels'] = cudf.Series(self._unique_labels)
+        if self._fit_status_ == 0:
+            state['_dual_coef_'] = self._dual_coef_
+            state['_support_'] = self._support_
+            state['_support_vectors_'] = \
+                self._support_vectors_
+            if self._n_classes > 0:
+                state['_unique_labels'] = \
+                    self._unique_labels
```

Comment:
This parameters would already be in the dict, since doing this `self._dual_coef_` is essentially accessing `__dict['_dual_coef_]` so this lines can be removed entirely